packages/shared/styles/styleUtils/modifiers/modifierUtils.ts

```diff
@@ -22,18 +29,30 @@ const spaceSizes = {
   none: 0
 };
 
-// TODO: add ability to use "horizontal" and "vertical"
-// to do left/right or top/bottom padding
 export const boxSpacing = (
   property: "margin" | "padding",
   side: BoxSides,
   spaceSize?: SpaceSizes
 ) => {
   const size = spaceSize || "m";
 
-  if (side === "all") {
-    return `${property}: ${spaceSizes[size]} !important;`;
-  } else {
-    return `${property}-${side}: ${spaceSizes[size]} !important;`;
+  switch (side) {
```

Comment:
I originally made this update to apply padding from standard spaces to the top and bottom of the button. When I reviewed with Lily, she told me the design team is intentionally deviating from standard spacing for button padding (currently 10px vertical, 18px horizontal).

Instead of removing this code I already wrote, I kept it in for future work